eng/Versions.props

```diff
@@ -124,7 +124,7 @@
   <PropertyGroup>
     <VersionToolsVersion>2.2.0-beta.19072.10</VersionToolsVersion>
     <DotnetDebToolVersion>2.0.0</DotnetDebToolVersion>
-    <MicrosoftNETTestSdkVersion>15.8.0</MicrosoftNETTestSdkVersion>
+    <MicrosoftNETTestSdkVersion>16.8.0-release-20200828-02</MicrosoftNETTestSdkVersion>
```

Comment:
The last time this was touched was the migration to Arcade: https://github.com/dotnet/installer/pull/140. It's apparently test-only, which could explain why it's behind the coherent product:

https://github.com/dotnet/installer/blob/b303c1b877bd3798f02d4469a3c4d50df33196e1/eng/Versions.props#L123-L127

It's not used in dotnet/installer. I think it's used to override this Arcade default: https://github.com/dotnet/arcade/blob/e11b8486ad155fab13e788a268755c0ca60d3609/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L81

If it's necessary to use an old version, we'll have to figure out how to move this prop out of the file (or depend on a different nupkg out of `vstest`, but that seems more fragile).